app/views/application_letters/_application_selective.html.erb

```diff
@@ -1,4 +1,4 @@
-<% if can? :edit_applicants, Event and not (application_letter.event.application_status_locked == true) %>
+<% if (can? :edit_applicants, Event) && (application_letter.event.phase == :selection) %>
```

Comment:
Oh, I forgot that sending emails/locking puts the evnt in execution phase
Yes, you are correct